### PR TITLE
Save data in browser

### PR DESCRIPTION
**Pull request by HishamAlmannai**
For userStory #10

### CODE REVIEW

**Review by anitaschiller — APPROVED** [review]
Looks good to me. If you remove the `persist` test it's approved! 😊 
Nevertheless I like that you are experimenting with tests a lot 👍

File: src/components/TaskList/TaskList.spec.js [hunk 1]
```diff
@@ -20,6 +21,20 @@ describe('TaskList', () => {
 		const buttonItem = screen.getAllByRole('button', { name: /delete/i });
 		userEvent.click(buttonItem[0]);
 
+		const listItemAfter = screen.getAllByRole('listitem');
+		expect(listItemAfter).toHaveLength(4);
+	});
+	it('renders five tasks and removes one and restarts the browser', () => {
```
[the remark this answers is not in the document]
Comment (HishamAlmannai) [reply src/components/TaskList/TaskList.spec.js:27]: Nice catch. I was wondering, if somebody finds out  😄

File: src/components/TaskList/TaskList.spec.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
+import React from 'react';
 import { userEvent } from '@storybook/testing-library';
-import '@testing-library/jest-dom';
 import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import TaskList from './TaskList';
 
 describe('TaskList', () => {
```
[the remark this answers is not in the document]
Comment (HishamAlmannai) [reply src/components/TaskList/TaskList.spec.js:7]: Yes, thanks to @aclaes I have. For testing I am using a other array. It stored in __mocks__>zustand.js

**Comment by NadineGrosskreuz** [comment]
Hey Hisham,
looks good, but, when I uncheck or check a task, it isn't saved when reloading the page.

**Review by NadineGrosskreuz — APPROVED** [review]
Looks good!

**Pull request merged**